spec/spec_helper.rb

```diff
@@ -21,11 +21,17 @@
     FactoryBot.reload
     ActionMailer::Base.deliveries.clear
 
+    stub_request(:get, "http://api.postcodes.io/postcodes/HA72BL").
+      to_return(status: 200, body: File.read("#{Rails.root}/spec/fixtures/postcode_conversion_payload.json"))
+
+    stub_request(:get, "http://api.postcodes.io/postcodes/notavalidpostcode").
+      to_return(status: 404, body: File.read("#{Rails.root}/spec/fixtures/postcode_error_conversion_payload.json"))
+
```

Comment:
You should stub this in the spec file rather than stubbing it globally here in the spec_helper